### ISSUE TITLE
Feature Request for Automatic Date Update

### ISSUE THREAD

**Issue by cl0udwalk3r** [ronkok/Hurmet#46]
Is it possible to add a feature like $PAGE for Dates that updates the date automatically when we save or print our calculations?

**Comment by ronkok** [comment]
I'll look into it. I'm currently working on improvements to print pagination. That turns out to be pretty complex, so it will be a couple of weeks before I get to this request.

When I worked with Microsoft Word, I generally found `$SAVEDATE` to be more useful than `$DATE`. Do you have a use case for `$DATE`?

**Comment by ketansahugit** [comment]
@ronkok  Hi, I have already started working on this. is that ok with you?

**Comment by ronkok** [comment]
@ketansahugit 

First, we should agree on the scope of work. I can easily see the value of a `$SAVEDATE` feature, but I do not see much value in a `$DATE` feature. I do not want to have to maintain a feature that has little value.

Second, I'm not sure this is the best moment. I am making significant changes to the pagination code and I anticipate merge conflicts if you write the `$SAVEDATE` feature now. Why don't you check in with me in about a week?

**Comment by cl0udwalk3r** [comment]
Hi everyone, i did mean to say $SAVEDATE , i agree $DATE doesn't do much.

**Comment by ketansahugit** [comment]
@ronkok 

Agreed. Let me know when you want the $SAVEDATE feature implemented. If I’m free at that time, I’ll look into it.

**Comment by ronkok** [comment]
Hurmet now has date math. You can create an automatic `savedate` field this way:

1. Save a Hurmet document. Only in the last 24 hours has Hurmet written the save date into the document metadata.
2. Create a Hurmet calculation field (**Alt+C**) and write `savedate() = @@`.
3. Adjust the date format to your taste using the menu choice at `Doc | Date Format…`.

Such a field can be created anywhere in a Hurmet document. If it is in a print header, it will appear on each printed page.
